pkg/net/libp2p/key.go

```diff
@@ -63,10 +67,11 @@ func networkPublicKeyToOperatorPublicKey(
 ) (*operator.PublicKey, error) {
 	switch publicKey := networkPublicKey.(type) {
 	case *libp2pcrypto.Secp256k1PublicKey:
+		ecdsaPublicKey := (*secp.PublicKey)(publicKey).ToECDSA()
 		return &operator.PublicKey{
 			Curve: operator.Secp256k1,
-			X:     publicKey.X,
-			Y:     publicKey.Y,
+			X:     ecdsaPublicKey.X,
+			Y:     ecdsaPublicKey.Y,
```

Comment:
And once we convert to `btcec`, we can:
```suggestion
			X:     btcecPublicKey.X(),
			Y:     btcecPublicKey.Y(),
```

Reply:
Done in dbcc2dfc6ab514c423f55f42a04a4baf03395f3c.